### DISCUSSION TITLE
Unsetting (emptying) a field

### DISCUSSION THREAD

**Discussion by protofolius** [comictagger/comictagger#527] [Q&A] [answered]
Whenever I try to unset a field through CLI command , it gives me an error 

`invalid parse_metadata_from_string value: 'issue='`

I tried also another field `"volume="` , but same error occurs .

I am using v1.6.0a5

**Comment by lordwelch** [comment, marked as answer]
I didn't notice I broke that in this release, it'll be fixed in the next alpha

**Comment by protofolius** [comment]
Bughunter and Squatter working together ;)

**Comment by protofolius** [comment]
This works again . Thank you .
